docs/insights/space-level-insights.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Space Level Insights
+description: Space level insights
+position: 140
+---
+
+Space level insights provide actionable DORA metrics for more complex scenarios across projects, project groups, environments, or tenants. This enables managers and decision-makers to get far more insight into the DevOps performance of their organization in line with their business context, such as team, portfolio, or platform.
+
+:::Information
+Space level sights requires and [enterprise subscription](https://octopus.com/enterprise). Contact your Octopus sales for a trial.
+:::
+
+To get started with space level insights choose the `Insights` item from the top Menu. From there you can create one or more reports.
+
+We recommend creating a separate reports for each purpose. Typically a report would be configured in one of the following ways:
+- One project or a small set of closely related project and all the relevant environments
+- Many projects and only similar environments (e.g. Production-like environments)
+
+If there are many unrelated projects and environments in a report, it will be hard to see any trends.
+
+## Settings
+
+### Time Zone
+
+The time zone of a report affects which day data is counted in when aggregating. The time zone should be chosen to reflect the most
+common understanding of what constitutes "midnight" for the team.
+
+### Projects
+
+Projects can be added to a report either by adding a project group, or individual projects. If a selected project already exists in a 
+selected group, it will not be double counted. There must be at least one project selected, directly or indirectly, for the report to generate data.
+
+:::info
+Only data from the default channel for each project is shown. This is to avoid pre-release and prior-version channels from skewing the data.
+:::
+
+### Environments
+
+Environment groups are a way to logically group environments so that they can be reported on together. In this section, create a group for each logical group of environment and then add the relevant environments to the group. By doing this you are able to compare the data between these groups using the split function. There must be at least one environment group with one environment for the report to generate data.
```

Comment:
```suggestion
Environment groups are a way to logically group environments so that they can be reported on together. In this section, create a group for each logical group of environments and then add the relevant environments to the group. By doing this you are able to compare the data between these groups using the split function. There must be at least one environment group containing one environment for the report to generate data.
```